tst/com/amazon/corretto/crypto/provider/test/AesTest.java

```diff
@@ -400,11 +504,6 @@ public void test_getOutputSize_encrypt() throws Throwable {
 
         // Allows room for the final tag
         assertEquals(12345 + 16, sneakyInvoke_int(spi, "engineGetOutputSize", 12345));
-
-        // Allows room for a partial buffered block
-        sneakyInvoke(spi, "engineInit", Cipher.ENCRYPT_MODE, key,
-                new GCMParameterSpec(12 * 8, randomIV()), rnd);
-        assertEquals(12345 + 15, sneakyInvoke_int(spi, "engineGetOutputSize", 12345));
```

Comment:
Just wondering why we can delete this? Since it is not immediately obvious to me I figured I'd as while I'm reading :)

Reply:
This is actually testing for the buggy behavior. Since the buggy behavior was never actually required, we are removing the test.